packages/@okta/i18n/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "_args": [
     [
-      "@okta/i18n@35.2.0",
-      "/Users/mauricio.castillosilva/okta/okta-signin-widget"
+      "@okta/i18n@35.5.0",
+      "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@35.2.0",
-  "_id": "@okta/i18n@35.2.0",
+  "_from": "@okta/i18n@35.5.0",
+  "_id": "@okta/i18n@35.5.0",
   "_inCache": true,
+  "_installable": true,
```

Comment:
this is another I'm not sure about, but if it passes Travis we should be good.

Reply:
It's green :)